oemer/build_system.py

```diff
@@ -114,27 +117,27 @@ def __repr__(self):
 
 
 class Measure:
-    def __init__(self):
-        self.symbols = []  # List of symbols
-        self.double_barline: bool = None
+    def __init__(self) -> None:
+        self.symbols: List[Any] = []  # List of symbols
+        self.double_barline: Union[bool, None] = None
         self.has_clef: bool = False
         self.clefs: list[Clef] = []
         self.voices: list[NoteGroup] = []
         self.sfns: list[Sfn] = []
         self.rests: list[Rest] = []
-        self.number: int = None
-        self.at_beginning: bool = None
-        self.group: int = None
+        self.number: Union[int, None] = None
+        self.at_beginning: bool = None  # type: ignore
+        self.group: Union[int, None] = None
 
-        self.time_slots: list[object] = []
-        self.slot_duras: np.ndarray = None
+        self.time_slots: list[List[Any]] = []
```

Comment:
`list[List[Any]]` -> `List[List[Any]]`
Make sure to choose only one typing style. Don't mix use different typing symbol.

Reply:
Fixed that one and in 3 more files where I found something similar.